internal/cli/printer/resource_test.go

```diff
@@ -0,0 +1,66 @@
+package printer
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestResourcePrinterWithJsonPrintsJsonWhenJsonOutputFormatIsSelected(t *testing.T) {
```

Comment:
Can we adjust the tests in the approach to how it is added in other places? I

Reply:
Adjusted